<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RssReader</class>
 <widget class="QWidget" name="RssReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USense RSS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resource.qrc">
    <normaloff>:/images/USenseIcon.png</normaloff>:/images/USenseIcon.png</iconset>
  </property>
  <widget class="QTreeWidget" name="treeWidget_Rsslist">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>250</width>
     <height>650</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTreeView::branch:has-siblings:!adjoins-item {
     border-image: url(:/images/rss/vline.png) 0;
 }

 QTreeView::branch:has-siblings:adjoins-item {
     border-image: url(:/images/rss/branch-more.png) 0;
 }

 QTreeView::branch:!has-children:!has-siblings:adjoins-item {
     border-image: url(:/images/rss/branch-end.png) 0;
 }

 QTreeView::branch:has-children:!has-siblings:closed,
 QTreeView::branch:closed:has-children:has-siblings {
         border-image: none;
         image: url(:/images/rss/branch-closed.png);
 }
 QTreeView::branch:open:has-children:!has-siblings,
 QTreeView::branch:open:has-children:has-siblings  {
         border-image: none;
         image: url(:/images/rss/branch-open.png);
 }
QTreeWidget
{
	background:rgb(222, 222, 222);
}</string>
   </property>
   <column>
    <property name="text">
     <string>我的订阅</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QWebView" name="webView_Rss" native="true">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>0</y>
     <width>290</width>
     <height>650</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWebView" name="webView_Passage" native="true">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>560</width>
     <height>650</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeWidget_Rsslist</tabstop>
  <tabstop>webView_Rss</tabstop>
  <tabstop>webView_Passage</tabstop>
 </tabstops>
 <resources>
  <include location="../../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
